js/pivx_shielding.js

```diff
@@ -107,15 +107,15 @@ export class PIVXShielding {
 
   /**
    * Loop through the txs of a block and update useful shield data
-   * @param {{tx: String[]}} blockJson - Json of the block outputted from any PIVX node
+   * @param {{txs: String[], height: Number}} blockJson - Json of the block outputted from any PIVX node
```

Comment:
why is the height needed?

Reply:
It is used in the function

Reply:
oh yeah I'm sleeping,  I thought that it was a new parameter of the function and not just a doc comment